reporter-opentsdb/src/test/java/io/ultrabrew/metrics/reporters/opentsdb/OpenTSDBHttpClientTest.java

```diff
@@ -86,6 +87,23 @@ public void testWriteFailsManualFlush() throws Exception {
     assertEquals(0, client.currentBatchSize);
   }
 
+  @Test
+  public void testNullStringsAndTagsValidation() throws Exception {
+    new Expectations() {{
+      
```

Comment:
If there are no expectations, just remove the entire block.

Reply:
done